docs/docs/platform/graphql/permissions.md

```diff
@@ -60,11 +60,10 @@ To use custom permission variables locally, add your claims to the `config.yml`
 
 ```
 auth:
-  jwt:
-    custom_claims: '{"organisation-id":"profile.organisation.id"}'
+  custom_claims: '{"organisation-id":"profile.organisation.id"}'
```

Comment:
this must be either the way it was before or:

```yaml
auth:
  jwt_custom_claims: ...
```

to match the `AUTH_JWT_CUSTOM_CLAIMS` [environment variable in Hasura Auth](https://github.com/nhost/hasura-auth/blob/main/src/utils/env.ts#L195).

Reply:
My bad.

> "the way it was before"

Didn't work for me. That's why I tried to help-out by fixing it. If this PR adds nothing, feel free to reject. No problem :)

> `jwt_custom_claims: …`

I'll try this locally.

Reply:
Ok, I'll close this PR for now. Please let me know if you can't get the custom claims to work and we can try to find out what's wrong. Maybe in a different issue.